components/dropdown/dropdown-opener-mixin.js

```diff
@@ -0,0 +1,145 @@
+export const DropdownOpenerMixin = superclass => class extends superclass {
+
+	static get properties() {
+		return {
+			isDropdownOpener: {

```

Comment:
I think we should follow the naming convention of omitting the `is` off of this boolean.